src/zeit/push/urbanairship.py

```diff
@@ -275,9 +240,11 @@ def product_config(self):
 def set_push_news_flag(context, event):
     push = zeit.push.interfaces.IPushMessages(context)
     for service in push.message_config:
+        # Just ignore channels for now
```

Comment:
Nach Rücksprache mit Thomas kann dass sogar gelöscht werden. Die CP, die mit solchen Artikeln gefüllt wird hat nur den Zweck, dass damit für alte Appversionen ein Link auf die CP gepusht werden kann (aus irgendwelchen Gründen konnten dort keine Links auf die Artikel selbst gepusht werden). Diese alten Versionen sind aber wohl kaum noch in Umlauf und es gibt auch für diese einen Workaround mit der neuen Templatelogik.